src/Akka.Streams.Kafka/Helpers/ExceptionExtensions.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using Confluent.Kafka;
+
+namespace Akka.Streams.Kafka.Helpers
+{
+    internal static class ExceptionExtensions
+    {
+        public static string GetCause(this Exception ex)
+            => ex switch
+            {
+                KafkaException {InnerException: null} ke => ke.Error.Reason,
+                KafkaException {InnerException: { }} ke => ke.InnerException.Message,
```

Comment:
When KafkaException has an inner exception, its Reason property defaults to returning the inner exception, including the stack trace, instead of the actual reason itself, making logs unreadable.

Reply:
Nice catch